cluster/charts/crossplane/templates/secret.yaml

```diff
@@ -10,4 +11,36 @@ metadata:
   name: webhook-tls-secret
   namespace: {{ .Release.Namespace }}
 type: Opaque
+{{- end }}
+{{- if $externalSecretStoresEnabled }}
+---
+# The reason this is created empty and filled by the init container is that it's
+# mounted by the actual container, so if it wasn't created by Helm, then the
+# deployment wouldn't be deployed at all with secret to mount not found error.
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ess-ca-certs
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+---
+# The reason this is created empty and filled by the init container is that it's
+# mounted by the actual container, so if it wasn't created by Helm, then the
+# deployment wouldn't be deployed at all with secret to mount not found error.
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ess-server-certs
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+---
+# The reason this is created empty and filled by the init container is that it's
+# mounted by the actual container, so if it wasn't created by Helm, then the
+# deployment wouldn't be deployed at all with secret to mount not found error.
```

Comment:
This comment seems to be valid only for `ess-client-certs`. So, we should either remove other secrets from here or still have them to be consistent but update the comment accordingly.